src/Command/GarbageCollector/AssignmentGarbageCollectorCommand.php

```diff
@@ -108,9 +108,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $successMessage = $numberOfCollectedAssignments !== 0
                 ? sprintf(
-                    "Total of '%s' stuck assignments were successfully marked as '%s'.",
-                    $numberOfCollectedAssignments,
-                    Assignment::STATE_COMPLETED
+                    "Total of '%s' stuck assignments were successfully collected.",
+                    $numberOfCollectedAssignments
                 )
                 : 'Nothing to update.';
```

Comment:
Just a proposal: maybe it'd be better for readability to split ternary operator?

Reply:
Can't figure out a better way to display it unfortunately, and I don't want to shorten the message for code style purposes